docs/source/import_export/main.rst

```diff
@@ -9,30 +9,29 @@ Export
 
 Use ``verdi export create`` to export a selection of nodes, codes, groups, users, or computers.
```

Comment:
I guess, for completeness, this could be and/or as well?
```suggestion
Use ``verdi export create`` to export a selection of nodes, codes, groups, users, and/or computers.
```